jest.config.mjs

```diff
@@ -7,6 +7,8 @@
 
 import {fileURLToPath} from 'url';
 
+process.env.TZ = 'UTC';
```

Comment:
While I trust you about the dev container changes (I saw the deprecation messages that some options should be changed), I'm curious about why this is necessary.

Reply:
I understand. Well, I cannot claim that it's always the case when the output is blank (although I will do some research about it tomorrow).

In the worst case, we can catch and print the error in a more generic way, like:

Failed to fetch update time for one or more files

Reply:
This is how we run jest today:

```json
{
  "scripts": {
    "test": "cross-env TZ=UTC jest" 
  }
}
```

But the VS Code Jest Extension does not read the npm scripts to figure out the proper jest command line to use. So, I could either configure the extension to run as `npx cross-env TZ=UTC jest`, or I could do this. This way, we don't duplicate the code.

About using `globalSetup`: adding it here makes it function more like it was before. But I do agree, `globalSetup` should be the correct place (although it will get executed for all test suites rather than being executed only at the begin).

Also, there isn't a globalSetup in the repo from what I saw (I would have to create one).

Please let me know how I should proceed.

Reply:
Just create a `globalSetup`, I think. But in the near future, we would also need to turn on `--experimental-vm-modules`, so I'm not sure what's the best direction... I'm okay to merge with this change and figure out later how to do it properly.